index.d.ts

```diff
@@ -65,7 +65,7 @@ export class SourceMap {
 
 export class Bundle {
   constructor(options?: BundleOptions);
-  addSource(source: MagicString | { filename?: string, content: MagicString }): Bundle;
+  addSource(source: MagicString | { filename?: string, content: MagicString, ignoreList?: boolean }): Bundle;
```

Comment:
And maybe we could have better documentations in TSDoc

Reply:
Done.